packages/esm-service-queues-app/src/queue-table-metrics-extensions/finished-service-status.extension.tsx

```diff
@@ -0,0 +1,25 @@
+import { useConfig } from '@openmrs/esm-framework';
+import React from 'react';
+import type { ConfigObject } from '../config-schema';
+import StatusQueueMetrics from './status-queue-table-metric-card.component';
+
+interface FinishedServiceStatusQueueMetricsProps {
+  queueUuid: string;
+  statusUuid: string;
+}
+
+export default function FinishedServiceStatusQueueMetrics({
+  queueUuid,
+  statusUuid,
+}: FinishedServiceStatusQueueMetricsProps) {
+  const config = useConfig<ConfigObject>();
+
+  if (!config?.concepts?.finishedServiceQueueStatusConceptUuid) {
+    console.error(new Error('No concept UUID found for Queue Status = Waiting for transfer'));
+    return null;
+  }
+
+  return (
+    <StatusQueueMetrics queueUuid={queueUuid} statusUuid={config?.concepts?.finishedServiceQueueStatusConceptUuid} />
+  );
+}
```

Comment:
I don't understand the point of this file. How does this add any value above the existing StatusQueueMetrics component? Why does this both pull a status concept from config and take one in as a prop? It should only take one in as a prop, and that is exactly what the StatusQueueMetrics already does. Delete this file.